python/eos/plot/config.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2018 Frederik Beaujean
+# Copyright (c) 2018 Danny van Dyk
+#
+# This file is part of the EOS project. EOS is free software;
+# you can redistribute it and/or modify it under the terms of the GNU General
+# Public License version 2, as published by the Free Software Foundation.
+#
+# EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+# Place, Suite 330, Boston, MA  02111-1307  USA
+
+import matplotlib
+from matplotlib import rcParams
+
+matplotlib.use('pgf')
```

Comment:
You should add a note in the manual like this: Import eos before you do something like `import matplotlib.pyplot as plt` because eos sets its default plot style and a matplotlib backend. All options (except the backend) can be overwritten by updating `matplotlib.rcParams[...]`. Note that the default plots use latex to create labels and math expressions, so for this to work latex needs to be available on your system.

Reply:
There is no section yet in the manual that discusses the Python interface at all.
I just made a new issue #54 out of your comment.